cmd/alertmanager/main.go

```diff
@@ -51,6 +51,7 @@ func main() {
 	}
 	defer server.Shutdown()
 
+	server.HTTP.PathPrefix("/alertmanager/status").Handler(multiAM.GetStatusHandler())
```

Comment:
Is the extra `/alertmanager` prefix needed here? We don't have a colliding status page yet, right?